### PR TITLE
Set up users/reviews seeding

### PR DESCRIPTION
**Pull request by jnthnch**

### CODE REVIEW

File: database-mongodb/usersSeed.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+};
+
+function getRandomTravelerExperience() {
+  const travelerNum = Math.floor(getRandomArbitrary(0, 3));
+  return travelExperience[travelerNum];
+}
+
+function randomNumber(minimum, maximum, precision) {
+  const min = minimum === undefined ? 0 : minimum;
+  const max = maximum === undefined ? 9007199254740992 : maximum;
```
Comment (richchuynh) [inline database-mongodb/usersSeed.js]: Following along the style of AirBnb, can the two ternary expressions be written as 

const min = minimum || 0;
const max = maximum || 9007199254740992 ;

File: database-mongodb/usersSeed.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  const precision1 = precision === undefined ? 0 : precision;
```
Comment (richchuynh) [inline database-mongodb/usersSeed.js]: Same as above

File: database-mongodb/usersSeed.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
+// SEEDING SCRIPT
+// "db:usersSeed": "node database-mongodb/usersSeed.js"
+
+const faker = require('faker');
+const db = require('./index.js');
+const User = require('./user.js');
+
+const sampleUsers = [];
+
+function getRandomArbitrary(min, max) {
```
Comment (richchuynh) [inline database-mongodb/usersSeed.js]: Seems like we have two random number generation functions, is it possible to use only one?

File: database-mongodb/usersSeed.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  const randomTravelExperience = getRandomTravelerExperience();
+  const randomReviewNumber = Math.floor(getRandomArbitrary(1, 3));
+  const userReviews = [];
+
+  for (let i = 0; i < randomReviewNumber; i += 1) {
+    const oneReview = createReview(username);
+    userReviews.push(oneReview);
+  }
+
+  const oneUser = {
```
Comment (richchuynh) [inline database-mongodb/usersSeed.js:90]: Code was kind of confusing to follow for this function. Can we move the user creation after all the properties were established? (Line 83)

File: database-mongodb/usersSeed.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  createUser();
+}
+
+function insertSampleUsers() {
+  User.create(sampleUsers)
+    .then(() => db.close())
+    .catch(err => err);
+}
+
+insertSampleUsers();
```
Comment (richchuynh) [inline database-mongodb/usersSeed.js:113]: AirBnB style, new line at end of code

**Review by richchuynh — CHANGES_REQUESTED** [review]

**Comment by richchuynh** [comment]
@jnthnch Left some comments

File: database-mongodb/usersSeed.js (hunk 1 above)
Comment (jnthnch) [reply database-mongodb/usersSeed.js]: Fixed, thanks

File: database-mongodb/usersSeed.js (hunk 2 above)
Comment (jnthnch) [reply database-mongodb/usersSeed.js]: Line 40,

Do you know why I can't use `precision` as a var name? There error I get is `Parsing error: Identifier 'precision' has already been declared`

File: database-mongodb/usersSeed.js (hunk 3 above)
Comment (jnthnch) [reply database-mongodb/usersSeed.js]: Fixed, thanks

File: database-mongodb/usersSeed.js (hunk 5 above)
Comment (jnthnch) [reply database-mongodb/usersSeed.js:113]: There must be an issue with my linter...I will add a new line at the end of code, then after a handful of saves, my linter removes the new line.

**Review by richchuynh — APPROVED** [review]
@jnthnch Thanks for making the changes, merging now

**Pull request merged**